helpGen.py

```diff
@@ -0,0 +1,116 @@
+#Imports
+import numpy as np
+from numpy import array
+from numpy.random import rand
+from numpy.random import shuffle
+from pickle import load
+from pickle import dump
+import re
+import os, sys, glob
+
+#Don't run these imports on your local machine!
+import tensorflow as tf
+#Keras imports
+from keras.layers import LSTM, Dense, Activation, Input
+from keras import optimizers
+from keras.models import Sequential
+from keras.layers.embeddings import Embedding
+from keras.preprocessing import sequence
+from keras.preprocessing.text import Tokenizer
+
```

Comment:
There should be a comment at the top of this file explaining:
- what this file does
- what this file requires (e.g. does it require particular files)
- what this file produces (e.g. does it write/create other files)
The information should be such that someone could read it and understand how to use/run the code